### PR TITLE
file2

### PR DESCRIPTION
**Pull request by chuanchyone1**
image

### CODE REVIEW

File: cypress/page/index-demoqa.ts [hunk 1]
```diff
@@ -1,8 +1,9 @@
 export { Login } from "./login-demoqa.page";
 export { PersonalInformation, UserInformation } from "./personal-information.page";
+//import "./commands";
```
Comment (danielgalvis98) [inline cypress/page/index-demoqa.ts]: Remove line

File: cypress/page/index-demoqa.ts [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
 
 Cypress.on('uncaught:exception', (err, runnable) => {
     // returning false here prevents Cypress from
     // failing the test
     return false
-  })
+  })
```
Comment (danielgalvis98) [inline cypress/page/index-demoqa.ts]: EOF

File: cypress/page/index.ts [hunk 3]
```diff
@@ -5,3 +5,5 @@ export { CartPage } from "./shopping-cart.page";
 export { InformationPage } from "./information.page";
 export { OverView } from "./overview.page";
 export { Finish } from "./checkout-complete.page";
+export { Login } from "./upload.page";
+
```
Comment (danielgalvis98) [inline cypress/page/index.ts]: Remove any blankspaces in this line

File: cypress/page/index-demoqa.ts (hunk 1 above)
Comment (chuanchyone1) [reply cypress/page/index-demoqa.ts]: I don't see this line in the project

image

Comment (chuanchyone1) [reply cypress/page/index-demoqa.ts]: update

File: cypress/page/index-demoqa.ts (hunk 2 above)
Comment (chuanchyone1) [reply cypress/page/index-demoqa.ts]: updated

File: cypress/page/index.ts (hunk 3 above)
Comment (chuanchyone1) [reply cypress/page/index.ts]: updated

**Review by danielgalvis98 — APPROVED** [review]

**Pull request merged**